lib/matplotlib/widgets.py

```diff
@@ -3789,7 +3790,8 @@ def onselect(verts):
     """
 
     @_api.make_keyword_only("3.7", name="useblit")
-    def __init__(self, ax, onselect, useblit=True, props=None, button=None):
+    def __init__(self, ax, onselect=lambda *args: None, useblit=True,
+                  props=None, button=None):
```

Comment:
Also, I am not fully sure that one would like to have a lambda as default value. Probably better in that case to use `None` and handle that it the code (either checking if it is `None` or set the lambda in the code in case of `None`).

Reply:
Hello! I wasn't sure about the decorator. But alright, I will implement the api decorator on the three functions. About the use of lambda, I used because it will try to run the function in the code, and if it is only set to None it will throw.

Reply:
I think you should look here:
https://github.com/matplotlib/matplotlib/blob/d464cbf5d665218401bab72879e229b1c5e0f616/lib/matplotlib/widgets.py#L2175-L2180
and
1. Add a default argument to `onselect` (onselect=None), no need to deprecate here
2. Add something like
``` python
if onselect is None:
    onselect = lambda...
self.onselect = onselect  // This is already there
```

Reply:
Ohh, Got it! I will do this! Thanks